docs/user/login-alternatives.md

```diff
@@ -0,0 +1,36 @@
+# Kubeapps Login Form
+
+By default, when first visiting Kubeapps, a login form is shown for the user to introduce a personal token for a specific service account:
```

Comment:
we can just say a Kubernetes API token, it doesn't have to be a service account token